demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
the metadata namespace has to be the namespace where the operator is running and watching for TPRs, so we actually can't use the namespace scope